lib/hammer_cli_katello/repository.rb

```diff
@@ -73,7 +73,7 @@ class InfoCommand < HammerCLIKatello::InfoCommand
       def extend_data(data)
         data["_redhat_repo"] = data["product_type"] == "redhat" ? _("yes") : _("no")
         data["_publish_via_http"] = data["unprotected"] ? _("yes") : _("no")
-        data["_sync_state"] = get_sync_status(data["sync_state"])
+        data["_sync_state"] = get_sync_status(data["last_sync"]["result"])
```

Comment:
I think you want `data["_sync_state"] = get_sync_status(data["last_sync"]["state"])` here